### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by am0**
One additional dad joke!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -76,3 +76,9 @@ It's funny 'cause it's true.
 **Q:** What's the second movie about a database engineer called?
 
 **A:** The SQL.
+
+---
+
+A programmer's wife tells them, "Run to the store and pick up a loaf of bread. If they have eggs, get two dozen."
+
+The programmer comes home with 12 loaves of bread.
```
Comment (chad3814) [inline readme.md]: should either be "24 loaves of bread" or "one dozen" on line 82

Comment (wesbos) [inline readme.md]: lol a+ code review.

**Comment by chad3814** [comment]
This is honestly my favorite programmer joke, it perfectly explains bugs.

File: readme.md (hunk 1 above)
Comment (am0) [inline readme.md]: Ah, my fault for writing from memory!

**Comment by wesbos** [comment]
😂

**Pull request merged**